vllm/utils.py

```diff
@@ -748,14 +761,21 @@ def is_pin_memory_available() -> bool:
     return True
 
 
+def _is_cuda_alike_stateless():
+    # Avoid circular import
+    from vllm.platforms import current_platform
```

Comment:
Currently no (`vllm.platform` only imports `vllm.logger` which in turn imports `vllm.envs`). Do you actively avoid importing other vLLM modules in `vllm.platform`? If so, I can use a regular import, otherwise IMO it's better to future-proof this.